app/test/tokumei/close_connection_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Tokumei.CloseConnectionTest do
+  use ExUnit.Case
+
+  alias Raxx.Response
+  alias Raxx.Request
+
+  use Tokumei.CloseConnection
+
+  def handle_request(%{method: :GET}, _), do: Response.ok("GET")
+
+  test "sends back connection: close if that was received" do
```

Comment:
Can we change these tests to run against running servers, connecting via a port? we can then test that the connection was closed as well as testing the headers are correct